addons/website/controllers/backend.py

```diff
@@ -26,8 +26,10 @@ def fetch_dashboard_data(self, website_id, date_from, date_to):
 
         current_website = website_id and Website.browse(website_id) or Website.get_current_website()
         multi_website = request.env.user.has_group('website.group_multi_website')
-        dashboard_data['websites'] = (multi_website and request.env['website'].search([]) or current_website).read(['id', 'name'])
-        for website in dashboard_data['websites']:
+        websites = multi_website and request.env['website'].search([]) or current_website
+        dashboard_data['websites'] = websites.read(['id', 'name', 'domain'])
```

Comment:
remove domain :)

Reply:
Indeed, leftover of previous version without `zip()`, done